lib/Model.php

```diff
@@ -456,8 +456,16 @@ public function assign_attribute($name, $value)
 		if ($value instanceof DateTime)
 			$value->attribute_of($this,$name);
 
+		if (array_key_exists($name,$this->attributes)) {
+			if ($this->attributes[$name] != $value) {
```

Comment:
if (array_key_exists($name,$this->attributes) && $this->attributes[$name] != $value) {}
...
else {
...
}